src/views/Languages/index.jsx

```diff
@@ -0,0 +1,265 @@
+    ...theme.mixins.gutters(),
+    minHeight: 60,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  link: {
+    textDecoration: 'none',
+  },
+  container: {
+    overflow: 'auto',
+    flexGrow: 1,
+    marginTop: 60,
+    padding: 2 * theme.spacing.unit,
+    [theme.breakpoints.up('md')]: {
+      marginLeft: DRAWER_WIDTH,
+      width: `calc(100% - ${DRAWER_WIDTH}px)`,
+    },
+  },
+  title: {
+    ...theme.mixins.gutters(),
+  },
+  button: {
+    color: theme.palette.common.white,
+    '& svg': {
+      fill: theme.palette.common.white,
+    },
+  },
+}))
+export default class Languages extends Component {
+  state = {
+    drawerOpen: false,
+    error: null,
+  };
+
+  handleDrawerToggle = () => {
+    this.setState({ drawerOpen: !this.state.drawerOpen });
+  };
+
+  handleBugInfoClick = memoizeWith(
+    id => id,
+    async id => {
+      try {
+        const {
+          data: { comments },
+        } = await this.props.client.query({
+          query: commentsQuery,
+          variables: { id },
+          context: { client: 'bugzilla' },
+        });
+
+        return comments[0].text;
+      } catch (error) {
+        this.setState({ error });
+      }
+    }
+  );
+
+  render() {
+    const {
+      classes,
+      match: {
+        params: { language },
+      },
+    } = this.props;
+    const { drawerOpen, error } = this.state;
+    const drawer = (
+      <Fragment>
+        <div className={classes.drawerHeader}>
+          <Typography variant="title">Skills</Typography>
+          <IconButton onClick={this.handleDrawerToggle}>
+            <CloseIcon />
+          </IconButton>
+        </div>
+        <Divider light />
+        <Sidebar activeItem={language} onItemClick={this.handleDrawerToggle} />
+      </Fragment>
+    );
+    const bugzillaData = this.props.bugzilla;
+    const bugs =
+      (bugzillaData &&
+        bugzillaData.bugs &&
+        uniqBy(
+          bugzillaData.bugs.edges.map(edge => edge.node).map(bug => ({
+            assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
+            project: bug.component,
+            tags: [
+              ...(bug.keywords || []),
+              ...extractWhiteboardTags(bug.whiteboard),
+            ],
+            summary: bug.summary,
+            lastUpdated: bug.lastChanged,
+            id: bug.id,
+            url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+          })),
+          'summary'
+        )) ||
+      [];
+
+    return (
+      <div className={classes.root}>
+        <AppBar position="fixed" className={classes.header}>
+          <Grid
+            container
+            className={classes.title}
+            alignItems="center"
+            spacing={8}>
+            <Hidden mdUp>
+              <Grid item>
+                <IconButton
+                  className={classes.button}
+                  size="large"
+                  onClick={this.handleDrawerToggle}>
```

Comment:
The drawer should be open by default on bigger screen sizes but closed on smaller ones.

_Example: taskcluster-web Desktop view_

![screen shot 2018-07-31 at 8 40 02 am](https://user-images.githubusercontent.com/3766511/43460008-5ac677b4-949d-11e8-95f3-e64db949fb99.png)

_Example: taskcluster-web mobile (smaller screen size) view_

![screen shot 2018-07-31 at 8 40 45 am](https://user-images.githubusercontent.com/3766511/43460042-750420c2-949d-11e8-98a2-599e5aaa5081.png)

Reply:
The screenshots are from `http://localhost:5080/provisioners`. But yes, the drawer behavior should be on Codetribute's home screen.